e2e/ansible/playbooks/resiliency/test-ctrl-failure/test-ctrl-failure.yml

```diff
@@ -103,15 +103,15 @@
        - name: 3) Replace storage-class in app yaml to use jiva based storage engine
          replace:
            path: "{{ result_kube_home.stdout }}/{{ percona_files.0 }}"
-           regexp: 'openebs-jiva-default'
+           regexp: 'openebs-standard'
```

Comment:
Let us be sure if this is the only place needing this change.

Reply:
yes. Here only we need these changes